src/Development/IDE/Core/FileStore.hs

```diff
@@ -165,17 +165,6 @@ fileStoreRules vfs = do
     getModificationTimeRule vfs
     getFileContentsRule vfs
 
-
--- | Notify the compiler service that a particular file has been modified.
---   Use 'Nothing' to say the file is no longer in the virtual file system
---   but should be sourced from disk, or 'Just' to give its new value.
-setBufferModified :: IdeState -> NormalizedFilePath -> Maybe T.Text -> IO ()
```

Comment:
Please keep this, we use it for tests in daml.

Reply:
I've rolled back all the dead code deletes